lib/activity_streams.ex

```diff
@@ -22,12 +22,9 @@ defmodule ActivityStreams do
   #   # Application.get_env(:activity_streams, :validators, %{})
   # end
 
-  def valid?(obj) do
-    false
-  end
-
+  # Decodes the json ActivityStream into its Type's corresponding struct
   def decode(json) do
-    case ActivityStreams.json().decode(json) do
+    case Poison.decode(json) do
```

Comment:
Not for this PR but it might be useful to move the act of decoding into a facade so we can let people bring their own JSON library instead of packing one for them.